hm_4/Task2(WSAD go)/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2_WSAD_go_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {   Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine("Убедитесь что включена !!!АНГЛИЙСКАЯ РАСКЛАДКА НА КЛАВИАТУРЕ!!!");
+            Console.Write("Введите символ для перемещения (W-вверх /S-вниз/А-влево/D-вправо): ");
+            char letter = Convert.ToChar(Console.ReadLine());
+            Console.WriteLine( "Вы ввели символ: " +letter);
+           if (  letter == 'w' || letter== 'W')
```

Comment:
I think switch operator looks better in this case.

Anyway, you can use `letter.ToLower()` instead of `letter == 'w' || letter== 'W'`